src/components/answer-genre/answer-genre.test.js

```diff
@@ -9,6 +9,7 @@ const answer = {
 };
 
 const userAnswer = true;
+const randomHandler = () => {};
```

Comment:
Давай переименуем эту заглушку здесь и в других тестах на:

```suggestion
const emptyHandler = () => {};
```

Дело в том что `randomHandler` плохо описывает что происходит внутри.

Reply:
Упс, не сразу подгрузились комменты в этом ПР, поэтому по аналогии с личным проектом переименовала в noopHandler, а потом уже заметила. Не критично?

Reply:
That's fine :)